Sean Larkin
Traceback (most recent call last):
  File "/home/raghavramky/Documents/GitHub_Project/try2.py", line 28, in <module>
    alpha = i.text
  File "/home/raghavramky/.local/lib/python2.7/site-packages/selenium/webdriver/remote/webelement.py", line 76, in text
    return self._execute(Command.GET_ELEMENT_TEXT)['value']
  File "/home/raghavramky/.local/lib/python2.7/site-packages/selenium/webdriver/remote/webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "/home/raghavramky/.local/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/raghavramky/.local/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document
  (Session info: chrome=69.0.3497.81)
  (Driver info: chromedriver=2.43.600233 (523efee95e3d68b8719b3a1c83051aa63aa6b10d),platform=Linux 4.15.0-20-generic x86_64)

[Finished in 29.3s with exit code 1]
[shell_cmd: python -u "/home/raghavramky/Documents/GitHub_Project/try2.py"]
[dir: /home/raghavramky/Documents/GitHub_Project]
[path: /home/raghavramky/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games]
